assets/css/sensei-course-theme/base.scss

```diff
@@ -1,9 +1,10 @@
-.sensei-course-theme__frame {
-
+.sensei-course-theme {
```

Comment:
Moved the variables to a level up, for now, so we can use it in other parts, like the notices.